workflows/export/timedmetadata_vod.go

```diff
@@ -0,0 +1,93 @@
+package export
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/bcc-code/bcc-media-flows/activities"
+	vsactivity "github.com/bcc-code/bcc-media-flows/activities/vidispine"
+	"github.com/bcc-code/bcc-media-flows/services/rclone"
+	"github.com/bcc-code/bcc-media-flows/services/telegram"
+	"github.com/bcc-code/bcc-media-flows/services/vidispine"
+	wfutils "github.com/bcc-code/bcc-media-flows/utils/workflows"
+	"github.com/bcc-code/bcc-media-platform/backend/asset"
+	"github.com/bcc-code/bcc-media-platform/backend/events"
+	"go.temporal.io/sdk/workflow"
+)
+
+type ExportTimedMetadataParams struct {
+	VXID string
+}
+
+type ExportTimedMetadataResult struct {
+	VXID string
+}
+
+// ExportTimedMetadata exports chapters to VOD as timedmetadata
+// After this flow, a job will be triggered in the BCC Media Platform to ingest the chapters.
+func ExportTimedMetadata(ctx workflow.Context, params ExportTimedMetadataParams) (*ExportTimedMetadataResult, error) {
+	logger := workflow.GetLogger(ctx)
+	logger.Info("Starting ExportTimedMetadata")
+
+	ctx = workflow.WithActivityOptions(ctx, wfutils.GetDefaultActivityOptions())
+
+	tempDir, err := wfutils.GetWorkflowTempFolder(ctx)
+	if err != nil {
+		return nil, err
+	}
+	outputDir := tempDir.Append("output")
+
+	var exportData *vidispine.ExportData
+	err = wfutils.Execute(ctx, vsactivity.Vidispine.GetExportDataActivity, vsactivity.GetExportDataParams{
+		VXID:        params.VXID,
+		Languages:   []string{"no"},
+		AudioSource: vidispine.ExportAudioSourceEmbedded.Value,
+	}).Get(ctx, &exportData)
+	if err != nil {
+		return nil, err
+	}
+
+	var timedMetadata *[]asset.TimedMetadata
+	err = wfutils.Execute(ctx, activities.Vidispine.GetChapterDataActivity, vsactivity.GetChapterDataParams{
+		ExportData: exportData,
+	}).Get(ctx, &timedMetadata)
+	if err != nil {
+		return nil, err
+	}
+
+	marshalled, err := json.Marshal(timedMetadata)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
`json.Marsall` is non-deterministic. There is (in pr.) a `wfultils.SafeMarsall` or something like that.